package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Let's also include a lock file

Reply:
ok, pnpm-lock.yaml is fine or package-lock.json

Reply:
I think package-lock.json is more preferable :+1: